content/en/docs/collector/resiliency.md

```diff
@@ -0,0 +1,229 @@
+---
+title: Resiliency
+description: How to configure a resilient OTel Collector pipeline
+---
+
+The OpenTelemetry Collector is designed with components and configurations to
+minimize data loss during telemetry processing and exporting. However,
+understanding the potential scenarios where data loss can occur and how to
+mitigate them is crucial for a resilient observability pipeline.
+
+## Understanding Collector Resilience
+
+The Collector's resilience primarily revolves around how it handles data when
+the configured endpoint (the destination for traces, metrics, or logs) becomes
+unavailable or when the Collector instance itself experiences issues like
+crashes.
+
+## Internal Queues (In-Memory Buffering)
+
+The most basic form of resilience built into the Collector's exporters is the
+**sending queue**.
```

Comment:
We generally only use bold for UI strings and similar. Please remove all instances or consider using subheadings for each of the items (in title casing, too).